v-next.sql

```diff
@@ -1,3 +1,4 @@
+\i ocd-3293.sql
 \i ocd-3110.sql
```

Comment:
These need to be removed. You should merge "staging" into your branch. It'll clean up the merge conflicts (temporarily anyway)

Reply:
Addressed as requested in below commit, thanks